mashina.Inc/src/main/webapp/User.jsp

```diff
@@ -0,0 +1,15 @@
+<html lang="en">
+<head>
```

Comment:
Plz rename the file with a lowercase letter according to jsp name convention https://www.ibm.com/support/knowledgecenter/en/SS73R8_9.4.0/com.ibm.help.custom.consolejsp.doc/JSP_Conventions_JSP_FileDir.html

Reply:
Use the following rules when choosing names for JSP files:
- Do not use any capital letters.
- Use underscores, not hyphens, to separate words.